app/src/test/java/com/kickstarter/libs/utils/ConfigUtilsTest.kt

```diff
@@ -32,13 +31,13 @@ class ConfigUtilsTest : KSRobolectricTestCase() {
         assertEquals(JSONArray(), ConfigUtils.enabledFeatureFlags(ConfigFactory.configWithFeatureEnabled("ios_native_checkout")))
 
         assertEquals(JSONArray(),
-                ConfigUtils.enabledFeatureFlags(ConfigFactory.configWithFeaturesEnabled(mapOf(Pair(FeatureKey.ANDROID_NATIVE_CHECKOUT, false),
```

Comment:
Was this feature key only used in tests? It might be a good idea to extract the checkout string

Reply:
Yep, it was only used in these tests since now it's no longer used in the actual app